### PR TITLE
log! scheme api docs; g_current_window() doxygen docs

### PR DESCRIPTION
**Pull request by graahnul-grom**

### CODE REVIEW

File: docs/scheme-api/geda-scheme.texi [hunk 1]
```diff
@@ -2055,6 +2055,8 @@ The @var{level} should be one of the symbols @code{error},
 @code{critical}, @code{warning}, @code{message}, @code{info} or
 @code{debug}.
 
+Note: if @var{level} is set to @code{'error}, main program will terminate.
```
Comment (vzh) [inline docs/scheme-api/geda-scheme.texi]: According to the style of previous sentences, the apostrophe symbol (') is unnecessary here :-)

File: schematic/src/g_window.c [hunk 2]
```diff
@@ -124,7 +124,7 @@ SCM_DEFINE (current_window, "%current-window", 0, 0, 0,
  * \brief Get the value of the #GschemToplevel fluid.
  * \par Function Description
  * Return the value of the #GschemToplevel fluid in the current dynamic
- * context.
+ * context. Never returns NULL.
```
Comment (vzh) [inline schematic/src/g_window.c]: I could even say: "Signals an error if there is no valid window fluid or the fluid value is NULL. Never returns NULL." WDYT?

File: docs/scheme-api/geda-scheme.texi (hunk 1 above)
Comment (graahnul-grom) [inline docs/scheme-api/geda-scheme.texi]: yes

File: schematic/src/g_window.c (hunk 2 above)
Comment (graahnul-grom) [inline schematic/src/g_window.c]: much better

**Pull request merged**